fineract-provider/build.gradle

```diff
@@ -76,7 +35,6 @@ apply plugin: 'com.github.andygoossens.gradle-modernizer-plugin'
 apply plugin: 'jacoco'
 apply plugin: "com.gorylenko.gradle-git-properties"
```

Comment:
Should we also remove the eclipse plugin as it doesn't work anymore (as discussed in the JIRA)?

Reply:
Yes, I was just waiting for you to confirm. Will do this right away.